exchanges/kline/kline_types.go

```diff
@@ -142,6 +142,7 @@ type ExchangeCapabilitiesSupported struct {
 
 // ExchangeCapabilitiesEnabled all kline related exchange enabled options
 type ExchangeCapabilitiesEnabled struct {
+	// Interval defines whether the exchange supports interval kline requests.
```

Comment:
The comment provided does not correspond to the name of the property. This discrepancy indicates that there is a mismatch between the identifier assigned to the property and the descriptive text provided in the comment. Such a deviation can cause confusion and make it difficult for other programmers or stakeholders to understand the intended purpose of the property. It is advisable to ensure that the comments accurately reflect the name and functionality of the associated properties to maintain clarity and avoid potential errors or misunderstandings.